modules/installation-configuration-parameters.adoc

```diff
@@ -2689,30 +2680,33 @@ ifdef::agent[]
 |The name of the failure domain.
 |String
 
+|platform:
+  vsphere:
+    failureDomains:
+      region:
+|If you define multiple failure domains for your cluster, you must attach the tag to each vCenter datacenter. To define a region, use a tag from the `openshift-region` tag category. For a single vSphere datacenter environment, you do not need to attach a tag, but you must enter an alphanumeric value, such as `datacenter`, for the parameter.
+|String
+
 |platform:
   vsphere:
     failureDomains:
       server:
-|The fully qualified domain name (FQDN) of the vCenter server.
-|An FQDN such as example.com
-endif::agent[]
+|Specifies the fully-qualified hostname or IP address of the VMware vCenter server, so that a client can access failure domain resources. You must apply the `server` role to the vSphere vCenter server location.
+|String
 
 |platform:
   vsphere:
     failureDomains:
-      topology:
-        datastore:
-|Specifies the path to a vSphere datastore that stores virtual machines files for a failure domain. You must apply the `datastore` role to the vSphere vCenter datastore location.
+      template:
```

Comment:
I think template is under 
platform:
  vsphere:
    failureDomains:
      topology:

Reply:
Thanks, Shang. On running `openshift-install explain installconfig.platform.vsphere.failureDomains`, I do not see an entry for `template. On running `openshift-install explain installconfig.platform.vsphere.failureDomains.topology` I do see an entry for `template`. I'll ask Joe to confirm.

Reply:
It is under topology, wonder what is up with explain

https://github.com/openshift/installer/blob/97feb644c79b2fcc0be2742a94394654a40f0ff6/pkg/types/vsphere/platform.go#L214

Reply:
Thanks, Joe and @sgaoshang . Updated the parameter and moved it along with other topology level parameter listings in the table.